cmake/AddFdbTest.cmake

```diff
@@ -371,11 +388,13 @@ function(package_bindingtester)
   set(tar_file ${CMAKE_BINARY_DIR}/packages/bindingtester-${CMAKE_PROJECT_VERSION}.tar.gz)
   add_custom_command(
     OUTPUT ${tar_file}
+    # AFAICT copy_bindingtester_binaries _should_ be sufficient, but for some
+    # reason ninja doesn't consider this target dirty if e.g. fdbcli changes.
+    DEPENDS copy_bindingtester_binaries ${outfiles} "${CMAKE_BINARY_DIR}/bindingtester.touch"
```

Comment:
same as above but I see that you executed the opposite of what I am thinking is necessary...